core/src/core/temporal/interfaces.py

```
@@ -0,0 +1,92 @@
+    error_message: str | None = None
+    metadata: dict[str, Any] = Field(default_factory=dict)
+    resilience_info: dict[str, Any] = Field(default_factory=dict)
+
+    @property
+    def is_success(self) -> bool:
+        """Проверяет, успешно ли выполнен этап."""
+        return self.status == "success"
+
+    @property
+    def is_failed(self) -> bool:
+        """Проверяет, завершился ли этап с ошибкой."""
```

Ruff (0.12.2)

Docstring contains ambiguous `с` (CYRILLIC SMALL LETTER ES). Did you mean `c` (LATIN SMALL LETTER C)?

(RUF002)